docs/presets/README.md

```diff
@@ -44,6 +44,9 @@ symbol = " "
 [golang]
 symbol = " "
 
+[haskell]
```

Comment:
Unfortunately, the preset system has been reworked somewhat in 1.4.0. The new location for preset files is `starship/docs/.vuepress/public/presets/toml`. Could you revert this to the version on master and then update the preset files in there?